src/commands/smartlog.rs

```diff
@@ -556,6 +571,7 @@ pub fn smartlog(
         &event_replayer,
         event_cursor,
         !show_hidden_commits,
+        *only_show_branches,
```

Comment:
It's been a *long* time since I had to deal w/ dereferencing, so I'm not totally clear on this, but the compiler told me I needed to and it seems to be working. I also tried `!!only_show_branches` which seemed to work, but this seemed closer to my (vague) understanding of Rust.

Reply:
That makes total sense now that you've explained it. Thank you!